### PR TITLE
Add new option `--only-pypi` 

### LINKED ISSUE #30: Automatically remove dependencies which are not available on PyPi
Should we remove from the dependency list the whole packages which are not present on ``PyPi`` when converting a conda package to a wheel?

Related to #29

cc: @scopatz

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Add new option `--only-pypi` which will remove all dependencies which are not available on PyPi.

It closes #30 

cc: @scopatz

### CODE REVIEW

File: conda_press/condatools.xsh [hunk 1]
```diff
@@ -687,8 +694,27 @@ class ArtifactInfo:
...
+    """Based on a list of dependencies this function will check if those
+    dependencies are on PyPi, if it is not available it will be removed.
+
+    :param set,list list_deps: List of dependencies as a string values
+    :return set: List of pakages present on PyPi
+    """
+    import requests
+    new_deps = set()
+    for pkg in list_deps:
+        response = requests.get(f"https://pypi.python.org/pypi/{pkg.lower()}/json")
```
Comment (scopatz) [inline conda_press/condatools.xsh]: I think PyPI also replaces `-` with `_` in package names.

File: conda_press/condatools.xsh [hunk 2]
```diff
@@ -687,8 +694,27 @@ class ArtifactInfo:
                 dep.clean()
 
 
+def get_only_deps_on_pypi(list_deps):
+    """Based on a list of dependencies this function will check if those
+    dependencies are on PyPi, if it is not available it will be removed.
+
+    :param set,list list_deps: List of dependencies as a string values
+    :return set: List of pakages present on PyPi
+    """
```
Comment (scopatz) [inline conda_press/condatools.xsh]: I usually prefer to use numpydoc format.  Is that OK with you?

**Comment by scopatz** [comment]
Thanks for putting this in!

File: conda_press/condatools.xsh (hunk 1 above)
Comment (marcelotrevisani) [reply conda_press/condatools.xsh]: Indeed! Good catch, I completely forgot about it!
I am going to modify this part and I will also create some tests to cover this scenario. Thanks!

File: conda_press/condatools.xsh (hunk 2 above)
Comment (marcelotrevisani) [reply conda_press/condatools.xsh]: Sure, no problem at all for me! I will change it

File: conda_press/condatools.xsh (hunk 1 above)
Comment (marcelotrevisani) [reply conda_press/condatools.xsh]: I was testing it, and it seems that ``pypi`` now accept the link to be with ``-`` or ``_``.
for example, with ``pytest-xdist``
https://pypi.org/pypi/pytest-xdist/json
https://pypi.org/pypi/pytest_xdist/json

both links return the metadata of ``pytest-xdist``

Comment (marcelotrevisani) [reply conda_press/condatools.xsh]: But I will replace the ``-`` by ``_``

**Comment by scopatz** [comment]
Thanks @marcelotrevisani!

**Pull request merged**